docker/zk-environment/Dockerfile

```diff
@@ -11,23 +11,24 @@ RUN add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debia
 RUN apt update; apt install -y docker-ce-cli
 
 # Install node and yarn
-RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -
 RUN apt-get install -y nodejs
-RUN npm install -g yarn
-RUN npm install -g ts-node
-
+RUN npm install -g yarn@1.22.17
+RUN npm install -g ts-node@10.9.1
 
 # Install required node packages
-RUN yarn global add aglio
-RUN yarn global add dredd
+RUN yarn global add aglio@2.3.0
+RUN yarn global add dredd@14.1.0
 
 # Install Rust and required cargo packages
 ENV RUSTUP_HOME=/usr/local/rustup \
     CARGO_HOME=/usr/local/cargo \
     PATH=/usr/local/cargo/bin:$PATH
 RUN curl https://sh.rustup.rs -sSf | bash -s -- -y
-RUN cargo install diesel_cli --no-default-features --features postgres
+RUN rustup install 1.69.0
+RUN cargo install diesel_cli  --version=1.4.0 --no-default-features --features postgres
 RUN cargo install --version=0.5.6 sqlx-cli
+RUN cargo install wasm-pack --git https://github.com/d3lm/wasm-pack --rev 713868b204f151acd1989c3f29ff9d3bc944c306
```

Comment:
Do we need this particular version of `wasm-pack`?

Reply:
Ill be removing this file from the PR.

Reply:
update